inductiva/resources/machines.py

```diff
@@ -52,6 +54,9 @@ def __init__(
             threads_per_core: The number of threads per core (1 or 2).
             spot: Whether to use spot machines.
             data_disk_gb: The size of the disk for user data (in GB).
+            disk_config: Disk configuration for the machine group. This config
+                includes disk size_gb and all the parameters needed for resizing
+                the disk. If provided will take precedence over data_disk_gb.
```

Comment:
I think they should be disjoint. Either one or the other, not both.

Reply:
I've added a check for that